sphinxext/mongodb_conf.py

```diff
@@ -331,6 +331,13 @@
         'prepend': True,
         'callable': False
     },
+    {
+        'name': 'k8s-op-om-setting',
```

Comment:
Hey @jwilliams-mongo , @jdestefano-mongo  -- this is rather a mouthful of a name and tag.  I think for usability, you all might want to shorten this.  

.. _k8s-op-om-setting:: foobar

then to link :k8s-op-om-setting:`foobar` . which I can see as inviting typos and forgetting what the setting name was.  

Also, for some references, you will have to prepend the tag so that people will have to do :k8s-op-om-setting:`~k8s-op-om-setting.foobar`

Reply:
good point @kay-kim. made these shorter. look good to you?